### DISCUSSION TITLE
Prerequisites for getting started

### DISCUSSION THREAD

**Discussion by parth721** [cncf/mentoring#755] [Q&A] [answered]
Hi 🙂
Do I need full knowledge of prometheus, kubernetes, OpenTelemetry, envoy all of them to start my opensource contribution as a beginnner or I can learn one of them & by identifying repo that uses the technology I learnt, start contributing ?

Have a great day ✌️

**Comment by redoC-A2k** [comment]
Yeah , thanks mate for asking this question . I am also very confused about what I should be knowing .

**Comment by nate-double-u** [comment, marked as answer]
Hi, you don't need 'full' knowledge as the mentorship program is a learning experience. 

I suggest checking out https://contribute.cncf.io/

You can also take a look at individual projects and see if they have any issues labelled "Good first issue" -- these are issues that projects have specifically noted for new comers and will provide extra help if needed (especially around setup and tooling).

You can use the new [Clotributor tool](https://clotributor.dev/search?foundation=cncf&good_first_issue=true&page=1) to help find these good first issues.

**Reply by redoC-A2k** [reply]
Thanks a lot for letting us know about "clotributor tool" . I will try to solve some good first issue , I hope they got merged .
